OpenRA.Mods.Common/ActorExts.cs

```diff
@@ -141,6 +141,10 @@ public static void NotifyBlocker(this Actor self, IEnumerable<CPos> positions)
 					return false;
 			}
 
+			// enemy unit or any other friendly unit is standing on it.
```

Comment:
Matye I can remove the line and move the comment to below and way "// including friendly units"